client/src/pages/SelectBoxScreen.js

```diff
@@ -1,7 +1,8 @@
-import React, { useState } from "react";
+// import React, { useState } from "react";
```

Comment:
Do you still need all the things that you have set to be comments? Otherwise you could as well remove them 😄

Reply:
Good point...🦾 i totally over saw that ...Thanks 🙂